apisix/core.lua

```diff
@@ -19,6 +19,8 @@ local local_conf = require("apisix.core.config_local").local_conf()
 
 local config_center = local_conf.apisix and local_conf.apisix.config_center
                       or "etcd"
+local etcd_version = local_conf.etcd.version == "v3" and "_v3" or ""
```

Comment:
all code in one line is not easy to read.

split it into multiple lines is better for this case.

Reply:
Sure, just which one would be better, if-else, or like `config_center` to split or case into next line